src/aks-preview/HISTORY.rst

```diff
@@ -11,6 +11,13 @@ To release a new version, please select a new version number (usually plus 1 to
 
 Pending
 +++++++
+
+2.0.0b3
```

Comment:
Please also update `VERSION` in `setup.py`.

Reply:
updated